src/common/Surface.js

```diff
@@ -157,6 +196,10 @@
         this._text
             .move({ x: (iconClientSize.width / 2 - menuClientSize.width / 2) / 2, y: yTitle })
         ;
+        this._surfaceButtons.forEach(function(button, idx) {
+            button.move({ x: (context._size.width / 2 - menuClientSize.width * 2) - 20 - buttonClientSize.width * idx, y: yTitle });
```

Comment:
Why -20?  Publish a gutter property for the gap between the menu and the buttons.